sdks/python/apache_beam/io/gcp/bigtable_io_write_pipeline.py

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unittest for GCP Bigtable testing."""
+from __future__ import absolute_import
+
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+class WriteBigtableOptions(PipelineOptions):
```

Comment:
Does it make sense to move this to `bigtable_io_write.py`?

It may also make sense to rename `bigtable_io_write.py` to `bigtable_io.py`, since we will be adding read to that file as well.